common/religion/religions/light_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_approved
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_shunned
+	doctrine = doctrine_life_magic_shunned
```

Comment:
Life magic could be ignored / accepted. Human came from practicing old way of druidism and most of them has syncretized with light religion, not to mention Gilnean Druids / Harvest-witches.

Reply:
Ngl, I was sure that I read something that says otherwise, but looks like I'm just misremembering things
Anyway, done!